test/unit/text.js

```diff
@@ -59,7 +59,7 @@
     skewY:                      0,
     charSpacing:                0,
     styles:                     [],
-    path:                       null,
+    path:                       undefined,
```

Comment:
i wonder if i need to make a backward compatibilitu statement or change somewhere.
The code never check if path is undefined, all it does is checking if path is truthy or not.
So null or undeined is the same also when reloading old data